pkg/api/unversioned/types.go

```diff
@@ -33,6 +38,8 @@ type TypeMeta struct {
 	// may reject unrecognized values.
 	// More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#resources
 	APIVersion string `json:"apiVersion,omitempty"`
+
+	Raw *RawData
```

Comment:
fwiw, go doesn't support reflection on non-exported fields, even within the same package, so this needs to be exported.